src/Prefabs/Vegetation/Tree/t_pinus_sylvestris_stump_01.et

```diff
@@ -0,0 +1,7 @@
+Tree : "{388AE316D09D0680}Prefabs/Vegetation/Core/Tree_Base.et" {
```

Comment:
To be fair, AR themselves messed this up, but I'd say we stick to CamelCase naming even if theirs are all lower case (probably from a mass import of trees)

So I'd say `T_Pinus_Sylvestris_Stump_01.et` etc. But this is only a mere suggestion ... if you want to leave it at that I won't complain

Reply:
The currently overridden trees like apple and plum tree all are all lowercase.